pkg/storage/stores/shipper/shipper_index_client.go

```diff
@@ -59,10 +59,10 @@ type indexClient struct {
 	stopOnce sync.Once
 }
 
-// NewShipper creates a shipper for syncing local objects with a store
-func NewShipper(cfg Config, storageClient client.ObjectClient, limits downloads.Limits,
-	tenantFilter downloads.TenantFilter, tableRange config.TableRange, registerer prometheus.Registerer, logger log.Logger) (series_index.Client, error) {
-	i := indexClient{
+// NewIndexClient creates a shipper for syncing local objects with a store
```

Comment:
should we also update this comment? since its creating boltdb store.
we could rename the file name as well or could be part of the package restructure follow-up